service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -44,7 +46,18 @@
     @Valid
     private final List<@NotNull(message = "You need to add details to applicant")
         Element<Applicant>> applicants;
-    private final List<Element<Respondent>> respondents1;
+    @NotNull(message = "You need to add details to respondents")
+    private final List<@NotNull(message = "You need to add details to respondents")Element<Respondent>> respondents1;
+
+    @Valid
+    private final Respondent getFirstRespondent() {
+        if (isEmpty(respondents1) || isEmpty(respondents1.get(0))) {
```

Comment:
Is there a chance that the value of the element could be null? If so then a check for that here might be useful instead of returning a null value.

Reply:
No but `isEmpty(respondents1.get(0)` is redundant. As we can assume if respondents is not empty it will have at least one item in the collection.